File: Regular/23-A.rb
#AtCoder Regular 23 A

def total_days year , month , day
    days = (365 * year) + (year/4).to_i - (year/100).to_i + (year/400).to_i
    days += (306*(month+1)/10).to_i
    days += day
    days -= 429

    return days
end

year = STDIN.gets.to_i
month = STDIN.gets.to_i
day = STDIN.gets.to_i

if month <= 2 then
    year -= 1
    month += 12
end

puts total_days( 2014 , 5 , 17 ) - total_days( year , month , day )

File: Regular/23/23-A.c
//AtCoder Regular 23 A
#include <stdio.h>

int total_days( int year , int month , int day ){
  int days = 0;
  days += (365 * year) + (year/4) - (year/100) + (year/400);
  days += (306*(month+1)/10);
  days += day;
  days -= 429;

  return days;
}

int main(void){
  int year;
  int month;
  int day;
  int tmp;

  scanf( "%d" , &year );
  scanf( "%d" , &month );
  scanf( "%d" , &day );

  if( month <= 2 ){
    year -= 1;
    month += 12;
  }

  tmp = total_days( 2014 , 5 , 17 ) - total_days( year , month , day );
  printf( "%d\n" , tmp );

  return 0;
}
